src/CodeReview.Orchestrator/CodeReview.Orchestrator.csproj

```diff
@@ -26,6 +26,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="ByteSize" Version="2.1.1" />
```

Comment:
I understand rational behind usage of this class. At the same time i think that this is extra dependency which we don't actually need. I would personally not introduce new Nuget package if I would like to have a separate type for file size rather than use `long`.

You may leave this package if you like. Decision belongs to you =)

Reply:
then I'll implement my own ByteSize class 😏